src/lwc/errRecordLog/__tests__/errRecordLog.test.js

```diff
@@ -0,0 +1,295 @@
+import { createElement } from 'lwc';
+import errRecordLog from 'c/errRecordLog';
+import { getObjectInfo } from 'lightning/uiObjectInfoApi';
+import { getNavigateCalledWith } from "lightning/navigation";
+import { registerSa11yMatcher } from '@sa11y/jest';
+
+import getData from '@salesforce/apex/ERR_Log_CTRL.getData';
+jest.mock(
+    '@salesforce/apex/ERR_Log_CTRL.getData',
+    () => {
+        return {
+            default: jest.fn(),
+        };
+    },
+    { virtual: true }
+);
+
+import { registerLdsTestWireAdapter } from '@salesforce/sfdx-lwc-jest';
+
+const getObjectInfoAdapter = registerLdsTestWireAdapter(getObjectInfo);
+
+const mockGetObjectInfo = require('./data/getObjectInfo.json');
+const mockGetData = require('./data/getData.json');
+const mockGetDataErrorLogs = require('./data/getDataErrorLogs.json');
+
+const RECORD_ID = "a0900000008MR9bQAG";
+const QA_LOCATOR_RECORD_PAGE = "breadcrumb Record View Page";
+const QA_LOCATOR_RECORD_SOBJECT_PAGE = "breadcrumb Record SObject Page";
+const QA_LOCATOR_NO_ITEM_MESSAGE = "text No Items Message";
+const QA_LOCATOR_DATATABLE = "datatable Logs";
+const QA_LOCATOR_NO_ACCESS_ILLUSTRATION = "illustration NoAccess";
+
+
+describe('c-err-record-log', () => {
+    let component;
+
+    beforeAll(() => {
+        registerSa11yMatcher();
+    });
+
+    beforeEach(() => {
+        component = createElement('c-err-record-log', {
+            is: errRecordLog,
+        });
+
+        getObjectInfoAdapter.emit(mockGetObjectInfo);
+    });
+
+    afterEach(() => {
+        clearDOM();
+        jest.clearAllMocks();
+    });
+
+    /***
+    * @description Verifies header is always displayed on the page
+    */
+    it('should display header', () => {
+        document.body.appendChild(component);
+
+        const header = component.shadowRoot.querySelector('h1');
+        expect(header).not.toBeNull();
+        expect(header.textContent).toBe('c.ERR_RecordLogTitle');
+    });
+
+
+    /***
+    * @description Verifies Error Log page elements when the record Id is specified
+    */
+    describe('on page render', () => {
+
+        beforeEach(() => {
+            component.recordId = RECORD_ID;
+            getData.mockResolvedValue(mockGetData);
+
+            document.body.appendChild(component);
+        });
+
+        it("should navigate to the record detail page", async () => {
```

Comment:
When defining the const, you use: 

"const QA_LOCATOR_RECORD_PAGE = "breadcrumb Record View Page";

either detail or record view seems fine, but should probably choose one and use that description everywhere.

Reply:
Thanks @npsp-reedestockton , the change has been made.